### DISCUSSION TITLE
Does adding a custom icon for the carousel icon only affect the fullscreen version?

### DISCUSSION THREAD

**Discussion by Tcox-int** [igordanchenko/yet-another-react-lightbox#296] [Q&A] [answered]
Hello,

I've added custom prevIcon and nextIcon svgs to the render function, but it only seems to change the arrows in the full screen carousel when I click into the image.  The svg icons on the actual carousel within the page are unchanged.  Do those use a different icon call?  I tried buttonNext and buttonPrev but those didn't work either.

```jsx
render={{
  buttonPrev: photos.length <= 1 ? () => null : undefined,
  buttonNext: photos.length <= 1 ? () => null : undefined,
  iconPrev: () => <LeftArrow />,
  iconNext: () => <RightArrow />,
}}
```

![image](https://github.com/user-attachments/assets/6a4abce4-99e1-4a87-9fe5-a7b7406353fa)
![image](https://github.com/user-attachments/assets/fd5eccb5-4f24-4cd8-85d2-5c62b2b05dc0)

**Comment by Tcox-int** [comment]
I also tried this

```jsx
render={{
  iconPrev: () => <LeftArrow />,
  iconNext: () => <RightArrow />,
  buttonPrev: photos.length <= 1 ? () => null : <LeftArrow />,
  buttonNext: photos.length <= 1 ? () => null : <RightArrow />,
}}
```

**Reply by igordanchenko** [reply]
Please share a minimal repro. You can use the following sandbox as a template.

https://stackblitz.com/edit/yet-another-react-lightbox-sandbox?file=src%2FApp.tsx

**Reply by igordanchenko** [reply, marked as answer]
@Tcox-int without seeing a reproducible example, I can only assume you are using 2 `Lightbox` instances, so you need to specify custom icons in both.

**Reply by Tcox-int** [reply]
And that was exactly it.  I didn't know there was a second instance.  That was the hint I needed. Thanks!
